cli/cephadm/cephadm.py

```diff
@@ -53,3 +53,21 @@ def prepare_host(self, **kw):
         cmd = f"{self.base_cmd} prepare-host {build_cmd_from_args(**kw)}"
 
         return self.execute(sudo=True, cmd=cmd)
+
+    def set_osd_flag(self, flag):
```

Comment:
If this is specific to osd, can this be moved out of cephadm class?
Refer: https://github.com/red-hat-storage/cephci/pull/2522/files#diff-a27dc52766cdabf382c0831b5d8a441ad8161b27673cd801cf4a16218708869e
Ignore if not valid

Reply:
Thanks @pranavprakash20 

As suggested , It will be good to move the method to osd lib.

As discussed once below PR merged, I will move the method to osd lib.

https://github.com/red-hat-storage/cephci/pull/2522/files#diff-a27dc52766cdabf382c0831b5d8a441ad8161b27673cd801cf4a16218708869e

Reply:
+1

Reply:
Added methods as mentioned above.